src/test/webdriver/geckodriver.rs

```diff
@@ -88,13 +89,21 @@ fn get_geckodriver_url(target: &str, ext: &str) -> String {
             .and_then(get_version_from_json)
             .and_then(save_geckodriver_version)
     };
-
+    let specified_geckodriver_version: Result<String, VarError> =
+        std::env::var("SPECIFIED_GECKO_DRIVER_VERSION");
     let geckodriver_version = if target::WINDOWS {
         log::info!(
             "[geckodriver] Windows detected, holding geckodriver version to {}",
             DEFAULT_WINDOWS_GECKODRIVER_VERSION
         );
         DEFAULT_WINDOWS_GECKODRIVER_VERSION.to_owned()
+    } else if specified_geckodriver_version.is_ok() {
```

Comment:
I would probably switch conditions `if target::WINDOWS` and `if specified_geckodriver_version.is_ok()` otherwise you wouldn't be able to specify the version on Windows.

Reply:
I wasn't sure because it seems the version for windows had to be 0.24.0.

Reply:
Done :)